statesman-server/docker-compose.yml

```diff
@@ -20,6 +20,7 @@ services:
     ports:
     - "8080:8080"
     - "8081:8081"
+    # - "8084:8084"
```

Comment:
Are we using compose file for production image? If not, we don't have to comment out stuff.

Reply:
According to me let's keep this commented and developer uncomment it as and when required.

Reply:
Why? Having the port exposed hurts nothing. OTOH this way, it adds a setp to the process

Reply:
Uncommented relevant code